.chloggen/split-components.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Based on our definition, this is NOT a breaking change.

Reply:
Since we're asking users of the component package to import a different module, isn't it breaking? The other splits were added to the changelog as breaking.

Reply:
> Since we're asking users of the component package to import a different module, isn't it breaking?

We don't ask, if they run `go mod tidy` is done (which they should to do the go.sum)

Reply:
With that in mind, we also made this decision https://github.com/open-telemetry/opentelemetry-collector/issues/6508#issuecomment-1309342642 

So we should be ok to do this.

Reply:
My point is, if we disagree and you believe this is a breaking change, lots of previous decisions should be revisited like the one I linked.

Reply:
Since with `component` module, users of the builder wouldn't be impacted, I think this is ok to mark as enhancement. Once this is merged I will do further testing w/ contrib to ensure this is true.

Reply:
Confirmed this is not a breaking change as per https://github.com/open-telemetry/opentelemetry-collector/actions/runs/3474306055/jobs/5807319018

Thanks for helping me work through this 👍🏻